packages/flutter/lib/src/services/text_input.dart

```diff
@@ -89,7 +89,8 @@ enum SmartQuotesType {
 class TextInputType {
   const TextInputType._(this.index)
     : signed = null,
-      decimal = null;
+      decimal = null,
```

Comment:
On looking at the `_configurationToJson` change, this doesn't have to change the public interface at all?

Reply:
I agree with you. I will try it on `_configurationToJson` and just change the JSON that is sent to the `engine`.

Reply:
I open a new [PR](https://github.com/flutter/flutter/pull/140356). Just added `forceMultiline` flag to JSON. @Renzo-Olivares @LongCatIsLooong please have a review.